rpc/client.go

```diff
@@ -377,10 +452,10 @@ func (c *Client) CallContextIgnoringLocalHandlers(ctx context.Context, result in
 
 	if c.router.routeRemote(method) {
 		client, err := c.getClientUsingCache(chainID)
-		if err != nil {
-			return err
+		if err == nil {
```

Comment:
I think you had undone this change in the other PR

Reply:
yeah, let me rebase this

Reply:
done